Casks/z/zen-browser@twilight.rb

```diff
@@ -0,0 +1,32 @@
+cask "zen-browser@twilight" do
+  arch arm: "aarch64", intel: "x86_64"
+
+  version "1.0.1-t.20"
+  sha256 arm:   "67c560a1568ec084b4a0b7c82787396f77da12954a77a9e194e084555c757ae9",
+         intel: "45f643373f3acb6a275f91718d0821f56001cbe21449822df2dc0ac4d9f33620"
+
+  url "https://github.com/zen-browser/desktop/releases/download/twilight/zen.macos-#{arch}.dmg",
+      verified: "github.com/zen-browser/desktop/"
+  name "Zen Twilight"
+  desc "Gecko based web browser"
+  homepage "https://zen-browser.app/"
+
+  livecheck do
+    url "https://github.com/zen-browser/desktop/releases/tag/twilight"
+    skip "No version information available"
```

Comment:
```suggestion
    url "https://api.github.com/repos/zen-browser/desktop/releases/tags/twilight"
    regex(/Twilight\s+build\s+-\s+v?(\d+(?:\.\d+)+(?:[.-][a-z]\.\d+)?)/i)
    strategy :json do |json, regex|
      match = json["name"]&.match(regex)
      next if match.blank?

      match[1]
    end
```

Upstream seems to use a fixed tag for twilight releases, and this approach allows us to extract the version information from the release's `name` attribute

@samford, could you take a look if you have the time? I feel like there might be a better approach I'm not seeing. Thank you!